localgov_moderngov_tpl.module

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Block\BlockPluginInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\localgov_moderngov_tpl\Constants;
+
+/**
+ * Implements hook_theme().
+ *
+ * - Default ModernGov page template.
+ */
+function localgov_moderngov_tpl_theme() {
+
+  $themes = [];
+  $themes[Constants::TPL_NAME] = [
+    'base hook' => 'page',
+  ];
+
+  return $themes;
+}
+
+/**
+ * Implements hook_theme_suggestions_hook_alter() for hook_theme_suggestions_page_alter().
+ *
+ * Suggests the ModernGov template for a particular node.
+ */
+function localgov_moderngov_tpl_theme_suggestions_page_alter(array &$suggestions, array $vars) {
+
+  $is_node = \Drupal::routeMatch()->getParameter('node');
+  if (!$is_node) {
+    return;
+  }
+
+  $is_moderngov_tpl_page = Drupal::service('localgov_moderngov_tpl.moderngov_path_predicate')->isModernGovTplPage();
+  if ($is_moderngov_tpl_page) {
+    $suggestions[] = Constants::TPL_NAME;
+  }
+}
+
+/**
+ * Implements hook_preprocess_hook() for hook_preprocess_html().
+ *
+ * - When in the ModernGov template page, applies appropriate classes on the
+ *   body tag.
+ */
+function localgov_moderngov_tpl_preprocess_html(&$vars) {
+
+  $is_moderngov_tpl_page = Drupal::service('localgov_moderngov_tpl.moderngov_path_predicate')->isModernGovTplPage();
+  if ($is_moderngov_tpl_page) {
+    $vars['attributes']['class'][] = 'node--localgov_modern_gov_page';
```

Comment:
Drupal coding standards requires hyphens/dashes instead of underscores to separate words in class names. Underscores are only for specifying elements level items inside (BEM) blocks - such as `node` with a `node__content` or `node__title` element inside it.

Since this class is on the body element, it should be page level, not node level. I suggest we rename it to `page--moderngov`